apps/crn-frontend/src/shared-research/state.ts

```diff
@@ -153,10 +153,14 @@ export const useSetResearchOutputItem = () => {
   );
 };
 
-export const useUserPermissions = (
-  researchOutputData: ResearchOutputResponse | undefined,
-): userPermissions => {
+export const useUserPermissions = ({
+  entity,
+  entityIds,
+}: {
+  entity: 'teams' | 'workingGroups';
+  entityIds: string[];
+}): UserPermissions => {
   const user = useCurrentUserCRN();
```

Comment:
We've discussed in the last RTM that maybe here we shouldn't get the user from Auth0, but I believe if we do this we should split into another ticket/PR since this one is already huge.